src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -142,6 +147,106 @@ public class Xthor : BaseCachingWebIndexer
             AddCategoryMapping(21, TorznabCatType.PC, "Logiciels Applis PC");
             AddCategoryMapping(22, TorznabCatType.PCMac, "Logiciels Applis Mac");
             AddCategoryMapping(23, TorznabCatType.PCMobileAndroid, "Logiciels Smartphone");
+
+            // Dynamic Configuration
+            ConfigData.AddDynamic("optionsConfigurationWarning", new DisplayInfoConfigurationItem(string.Empty, "<center><b>Available Options</b></center>,<br /><br /> <ul><li><b>Freeleech Only</b>: (<i>Restrictive</i>) If you want to discover only freeleech torrents to not impact your ratio, check the related box. So only torrents marked as freeleech will be returned instead of all.</li><br /><li><b>Specific Language</b>: (<i>Restrictive</i>) You can scope your searches with a specific language / accent.</li></ul>"));
```

Comment:
This should be removed, it's just duplicating information.

Reply:
Yes i could remove it, I just added for better understanding of available functions.